specs/dynamicdependencies/sample-b.1.md

```diff
@@ -0,0 +1,107 @@
+# Sample B.1 - HelloWorld console app using the Boostrap API
+
+At runtime, HelloWorld wants to use the Windows App SDK Framework 1.0-preview1 package, so it calls the Bootstrap API.
+
+## Win32
+
+```c++
+#include <iostream>
+
+#include <MddBootstrap.h>
+
+int main()
+{
+    // Initialize access to Windows App SDK
+    const UINT32 c_majorMinorVersion{ 0x00010000 };
+    PCWSTR c_versionTag{ L"preview1" };
```

Comment:
Just use `auto` here.

Reply:
auto == `const wchar_t *`
PCWSTR == `typedef _Nullterminated_ const wchar_t*`

Is SAL dead to us?